wiki-docs/docs/interfaces/network/exit-helper.md

```diff
@@ -0,0 +1,59 @@
+The `IExitHelper` interface is a helper contract that processes exits from stored event roots in `CheckpointManager`. It allows users to perform an exit for one or multiple events. This user guide will explain how to interact with the functions provided by the IExitHelper interface.
+
+## Functions
+
+### exit()
+
+This function allows you to perform an exit for one event.
+
+#### Parameters
+
+- `blockNumber` (uint256): The block number of the exit event on L2.
+- `leafIndex` (uint256): The leaf index in the exit event Merkle tree.
+- `unhashedLeaf` (bytes): The ABI-encoded exit event leaf.
+- `proof` (bytes32[]): The proof of the event inclusion in the tree.
+
+#### Usage
+
+To exit an event, call the `exit()` function with the required parameters:
+
+```solidity
+IExitHelper.exitHelperInstance.exit(blockNumber, leafIndex, unhashedLeaf, proof);
+```
+
+### batchExit()
+
+This function allows you to perform a batch exit for multiple events.
+
+#### Parameters
+
+- `inputs` (BatchExitInput[]): An array of BatchExitInput structs, where each struct contains the following fields:
```

Comment:
This function only has `BatchExitInput[] calldata inputs` parameter.

Reply:
i think that's ok, the struct elements just needed tabbing over ... however, please check as i'm not 100% sure what you meant, updated in upcming commit